HomeWork/src/main/java/io/FileLoggerConfiguration.java

```diff
@@ -0,0 +1,105 @@
+package io;
+
+import java.io.File;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class FileLoggerConfiguration {
+
+    String fileName = "NIO.txt";
+    private LoggingLevel loggingLevel;
+    private LoggingLevel[] arrayLevels;
```

Comment:
This array is redundant, remove it, and the parts that are logically related to it.